features/facebookImport/src/model/analyses/utils/analysis-status.js

```diff
@@ -1,14 +1,12 @@
-export function createSuccessStatus(analysisClass) {
+export function createSuccessStatus() {
```

Comment:
So I guess this is where you're switching the analysisClass out of the status?

Reply:
Yes, we no longer needed it.